src/pipecat/transports/daily/transport.py

```diff
@@ -104,6 +105,15 @@ class DailyInputTransportMessageUrgentFrame(InputTransportMessageUrgentFrame):
 
     participant_id: Optional[str] = None
 
+@dataclass
+class DailyUpdateRemoteParticipantsFrame(DataFrame):
+    """Frame to update remote participants in Daily calls.
+
+    Parameters:
+        remote_participants: See https://reference-python.daily.co/api_reference.html#daily.CallClient.update_remote_participants.
+    """
+
+    remote_participants: Optional[Any] = None
```

Comment:
In other parts of the code, we are mapping the remote_participants as: `remote_participants: Mapping[str, Any]`

Shouldn’t we do the same here ? I believe this would match with the docs.